packages/ubuntu_provision_test/lib/src/wizard_tester.dart

```diff
@@ -36,7 +39,7 @@ extension UbuntuWizardTester on WidgetTester {
 
   Future<void> jumpToPage(String route) async {
     final context = element(find.byType(WizardPage));
-    Wizard.of(context).jump(route);
+    await Wizard.of(context).jump(route);
```

Comment:
```suggestion
    unawaited(Wizard.of(context).jump(route));
```
should fix the screenshot tests (needs a `dart:async` import)